src/components/thead/thead-styles.jsx

```diff
@@ -2,23 +2,28 @@ import styleUtils from '../table/style-utilities';
 import color from '../../styles/color';
 
 const normal = theme => {
-  const { palette, mixins, typography } = theme;
+  const { mixins, typography } = theme;
 
   return {
     thead: {
       width: '100%',
       ...styleUtils.sizes(),
       ...mixins.basicBoxSizing,
       fontWeight: typography.fontWeightSemiBold,
-      borderColor: color.gray,
-      borderBottom: `2px solid ${palette.shades.dark.text.muted}`,
+      borderBottom: `2px solid ${color.grayDarker}`,
+      '& th': {
```

Comment:
`th `can't be ouside `thead`, is there need for this specificity?

Reply:
Prevents someone to sneak in other stuff inside thead and use the style if we just accepted any child